### PR TITLE
WIP: rvalle/feature docker

### PR DESCRIPTION
**Pull request by rvalle**
Docker builder based on  @insipx stale branch

I am currently testing it, but PR is WIP but in case anybody is interested to test it...

### CODE REVIEW

**Comment by rvalle** [comment]
The container is working and indexing the polkadot blockchain.
However it might be good idea to provide a fully working docker-compose file to help people get started with archiving a substrate blockchain.

**Comment by insipx** [comment]
Going to test this out later!

File: docker/build.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+set -e
+
+pushd .
+
+# The following line ensure we run from the project root
+PROJECT_ROOT=`git rev-parse --show-toplevel`
+cd $PROJECT_ROOT
+
+# Find the current version from Cargo.toml
+VERSION=`grep "^version" ./Cargo.toml | egrep -o "([0-9\.]+)"`
```
[the remark this answers is not in the document]
Comment (rvalle) [reply docker/build.sh:11]: good point. Yes, I noticed it failing to collect any version.

**Comment by rvalle** [comment]
As discussed, it would be nice that this Dockerfile generates the WASM modules with tracing required for substrate-archive be more capable to generate events.

Working on it.

**Comment by rvalle** [comment]
Discussed on the Chat and seems like automatic compilation of past WASM modules in batch mode is not feasible due to compiler errors :(

I think I will try to use a builder Dockerfile to assist in the creation of new WASM  modules while also just copy them for distribution into the node-archive docker.

**Pull request closed without merging**